src/buffer/buffer_map.rs

```diff
@@ -315,12 +315,15 @@ impl Accessed for BufferMap {
 mod tests {
     use crate::{prelude::*, testing::*, BufferMap};
 
-    #[derive(Clone, JoinedValue)]
+    #[derive(JoinedValue)]
     struct TestJoinedValue<T: Send + Sync + 'static + Clone> {
         integer: i64,
         float: f64,
         string: String,
         generic: T,
+        #[buffers(buffer_type = AnyBuffer)]
+        #[allow(unused)]
```

Comment:
I think it would make sense to pass a real value through this `AnyBuffer` and test that it comes out correctly on the other side instead of leaving it unused.